src/claims/bold/intellij/avro/idl/AvroIdl.bnf

```diff
@@ -70,11 +70,12 @@
 
 file ::= protocol_def*
 
-protocol_def ::= annotation? 'protocol' IDENTIFIER '{' declaration* '}' { pin = 2 }
+protocol_def ::= annotation* 'protocol' IDENTIFIER '{' declaration* '}' { pin = 2 }
 
 annotation ::= '@' IDENTIFIER '(' annotation_body ')' { pin = 1 }
```

Comment:
Agreed, good callout. I should have investigated what the more general possibilities were. Thanks for digging deeper!